src/locales/en/common.json

```diff
@@ -330,7 +330,7 @@
   "Invalid amount": "Invalid amount",
```

Comment:
NRTTL
When i switch back to correct application i still get this message

Reply:
You need to be sure that the WalletConnect configuration was set up with the right token details else you'd always get such errors

Reply:
Aha 👍